data/strings/strings.txt

```diff
@@ -17056,7 +17058,7 @@
 
   [transliteration_title]
     tags = android,ios
-    en = Transliteration into Latin
+    en = Transliterate into Latin
```

Comment:
I see Latin as a language, not an alphabet.   I would call the alphabet Roman.  However, wikipedia disagrees.

Still, I would say  "Transliteration into Latin alphabet" to make it clearer.